Fix alembic tooling

ttung opened the pull request:
### Description
Autogenerate requires the `include_schemas=True` to work.

#### Issue
[ch65314](https://app.clubhouse.io/genepi/stories/space/65314)

### Test plan
Generated some migrations.

lvreynoso commented on database_migrations/env.py:
```diff
@@ -1,6 +1,8 @@
 import os
 from logging.config import fileConfig
 
+from sqlalchemy import create_engine
```
looks like this appears twice now?

ttung replied:
Fixed.

Commit:
Fix alembic tooling

### Description
Autogenerate requires the `include_schemas=True` to work.

#### Issue
[ch65314](https://app.clubhouse.io/genepi/stories/space/65314)

### Test plan
Generated some migrations.
